js/gc-smart-data.js

```diff
@@ -336,6 +344,9 @@ window.GC = window.GC || {};
 
         if (boneAgeList && boneAgeList.length) {
             $.each(boneAgeList, function(i, o) {
+                if (o.hasOwnProperty("boneAgeMos") && o.hasOwnProperty("date") && o.hasOwnProperty("dateString")) {
```

Comment:
check o?

Reply:
$.each will guarantee that `o` always has a value .